src/DynamoDbModel.php

```diff
@@ -86,11 +86,38 @@ protected function setupDynamoDb()
 
     public function save(array $options = [])
     {
-        if (!$this->getKey()) {
-            $this->fireModelEvent('creating');
+        $create = !$this->exists;
+        if ($create) {
+            if ($this->fireModelEvent('creating')  === false) {
+                return false;
+            }
+        } else {
+            if ($this->fireModelEvent('updating') === false) {
+                return false;
+            }
+        }
+
+        if ($this->fireModelEvent('saving') === false) {
+            return false;
         }
 
-        return $this->newQuery()->save();
+        if ($this->usesTimestamps()) {
+            $this->updateTimestamps();
```

Comment:
Eloquent sets the value of the timestamps explicitly, and does not rely on the database to set values.  This would allow it to work with DynamoDB and the developer can just set the timestamp column names by overriding the UPDATED_AT and CREATED_AT consts.

We could modify the model the disable timestamps by default (they are currently enabled by default) to prevent any confusion.  Currently, we have to disable them explicitly in our model by setting `$timestamps = false;` ... this could be the default, with a note about it in the README?